Makefile

```diff
@@ -30,3 +26,23 @@ test :
 cover :
 	go test ./msgservice ./dbservice ./router ./formatting ./data ./regoservice ./routes -v -coverprofile=cover.out
 	go tool cover -html=cover.out
+
+develop :
```

Comment:
Postee UI has own Dockerfile. So Postee UI image should be released to Dockerhub some way. Also there should be some k8s support for Postee UI